packages/db/src/basedb.js

```diff
@@ -196,6 +196,26 @@ class BaseDB {
     return result;
   }
 
+  /**
+   * Inserts a single document to a collection.
+   *
+   * @param {string} modelName The model name, e.g. `platform.Content`.
+   * @param {object} document The update operations to be applied to the document.
```

Comment:
Change to `doc` to match the native driver and change comment to: `Document to insert.`